reliability.md

```diff
@@ -1,33 +1,45 @@
 # Reliability
 
-Once your site is up, how do you keep it that way? And what can you do if it goes down? In most cases, the same best practices apply to WordPress as with other web applications, but some differences and recommendations are detailed here.
+When we already have a site in operation, the work of hosting does not end. The machines can and do fail, so we need to have methods to check that it is working as well as a backup system.
 
-## Backups
+## Backup
 
-A WordPress site is composed of three (3) main components: Code WordPress core, zero (0) or more plugins, and one (1) or more themes Assets Typically images, documents and other user-upload files. This can also contain plugin or theme cache and/or configuration files as well. Database A set of tables containing your posts, pages, comments, links, settings, and other data.
```

Comment:
This is terse, but I like that it explains exactly the parts of a WordPress install, and I think that's a bit less clear in the rewrite below, because it doesn't talk about the parts outside of "files and database".

Maybe there's a way to reword this without removing it?